Shared/AppSecrets.swift.masque

```diff
@@ -15,7 +15,7 @@
 import ArcGIS
 
 extension String {
-    /// Your organization's ArcGIS Maps SDK [license](https://developers.arcgis.com/swift/license-and-deployment/) key.
+    /// Your organization's ArcGIS Maps SDK [license](https://developers.arcgis.com/swift/license-and-deployment/) string.
```

Comment:
You might want to replace the url with this https://developers.arcgis.com/swift/license-and-deployment/get-a-license/ 
It will get them to the License String part more quickly.